### ISSUE TITLE
ftprint does not print at krate

### ISSUE THREAD

**Issue by docb** [csound/csound#1447]
i assumed that the following code would print every second the provided table. (i made this example as small as possible)

```
<CsoundSynthesizer>

<CsOptions>
-odac
</CsOptions>

<CsInstruments>
sr        = 48000
ksmps     = 32
nchnls    = 2
0dbfs = 1
instr 1
  itab = p4  
  ktrig metro 1
  ftprint itab,ktrig
endin

</CsInstruments>

<CsScore>
f10 0 8 -2  0 1 2 3  4 5 6 7
i1 0 20 10
e

</CsScore>
</CsoundSynthesizer>
```
 but it is only printed once.

system: linux ubuntu

0dBFS level = 32768.0
--Csound version 6.15 (double samples) Mar  9 2021
[commit: e6c9ce1d1f20cb2a688402be089961f3583d2b37]
libsndfile-1.0.28
UnifiedCSD:  ftprint.csd
STARTING FILE
Creating options
Creating orchestra
closing tag
Creating score
rtaudio: ALSA module enabled
rtmidi: ALSA Raw MIDI module enabled
Elapsed time at end of orchestra compile: real: 0.002s, CPU: 0.002s
sorting score ...
	... done
Elapsed time at end of score sort: real: 0.002s, CPU: 0.002s
graphics suppressed, ascii substituted
0dBFS level = 1.0
orch now loaded
audio buffered in 256 sample-frame blocks
ALSA output: total buffer size: 1024, period size: 256
writing 512 sample blks of 64-bit floats to dac
SECTION 1:
ftable 10:
ftable 10:	8 points, scalemax 7.000
       _

      -
     _

    -
   .

  '
 .
________
new alloc for instr 1:
ftable 10:
   0: 0.0000 1.0000 2.0000 3.0000 4.0000 5.0000 6.0000 7.0000 

may be there is a trick but the manual says:
ktrig - The table will be printed whenever this value changes from 0 to possitive. Can be used together with metro to print at a given time interval. A value of -1 indicates to print at each k-cycle (default = 1)

**Comment by gesellkammer** [comment]
Thanks for the report. It was indeed a bug, a fix is on the way (see PR #1448)

**Comment by docb** [comment]
Thanks, i have applied the single change in ftprint and it works now.
